src/api/app/views/webui/package/_commit_item.html.haml

```diff
@@ -5,13 +5,17 @@
   - if user
     = image_tag_for(user, size: 20)
     = link_to(user_path(user)) do
-      = user.realname.present? ? "#{user.realname} (#{user})" : user
+      = realname_with_login(user)
   - else
     %i= commit['user'] || '-'
 
   - if commit['requestid']
     accepted
     = link_to("request #{commit['requestid']}", request_show_path(commit['requestid']))
+    - creator = BsRequest.find(commit['requestid']).creator
```

Comment:
This will barf an `ActiveRecord::RecordNotFound` if the request can't be found...

Reply:
True, fixed